packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -50,8 +51,11 @@ export class Prerendering {
             this.diesRenderInfo.push({
                 x: horizontalScale(die.x) + margin.right,
                 y: verticalScale(die.y) + margin.top,
-                fillStyle: this.calculateFillStyle(die, colorScaleMode),
-                opacity: this.calculateOpacity(die.value, highlightedValues),
```

Comment:
Does `opacity` need to stay on the `DieRenderInfo` type?

Reply:
no, the opacity was used in the rendering and converted to the alpha channel there, but we decided to do everything in the prerendering so I included that process here. I also removed the property from the DieRenderInfo type